### PR TITLE
Feat/game averages1 vs

### PR DESCRIPTION
**Pull request by vsanantone**
Fixing simple cov merge issues with .gitignore.

### CODE REVIEW

File: coverage/.resultset.json
Comment (EricBelongea) [inline coverage/.resultset.json:1]: Thank you for deleting these files.

**Review by EricBelongea — APPROVED** [review]
Good

File: lib/game.rb
Comment (zkevkev) [inline lib/game.rb:1]: This looks like a great start! I think we can refactor this a bit to get it down to a more manageable line count. Very nice job!

**Review by zkevkev — APPROVED** [review]
This looks like a great start! I think we can refactor this a bit to get it down to a more manageable line count. Very nice job!

**Review by billyburch — APPROVED** [review]

**Pull request merged**